source/material_model/visco_plastic.cc

```diff
@@ -362,11 +373,39 @@ namespace aspect
                              "instead of calculating the values through the specified thermal diffusivities, "
                              "densities, and heat capacities. ");
           prm.declare_entry ("Thermal conductivities", "3.0",
-                             Patterns::List(Patterns::Double(0)),
+                             Patterns::List(Patterns::Double (0.)),
                              "List of thermal conductivities, for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
                              "If only one value is given, then all use the same value. "
                              "Units: \\si{\\watt\\per\\meter\\per\\kelvin}.");
+          prm.declare_entry ("Use hydrothermal conductivity", "false",
```

Comment:
```suggestion
          prm.declare_entry ("Approximate hydrothermal cooling through thermal conductivity", "false",
```
Or something like that :)

Reply:
I think the term 'hydrothermal conductivity' can be improved upon. We adapt the thermal conductivity to approximate hydrothermal cooling.